.github/workflows/code_generator.yml

```diff
@@ -56,6 +56,10 @@ jobs:
         working-directory: ./frb_example/with_flutter
         run: flutter pub get
 
+      - name: Check DartApi version.
+        run: wget https://github.com/dart-lang/sdk/blob/master/runtime/include/dart_version.h?raw=true 
+          -O ./frb_rust/src/dart_api_dl/include/dart_version.h && git diff --exit-code
```

Comment:
nit: would be great to check the whole `include` folder

e.g. using clone with one commit https://stackoverflow.com/questions/31278902/how-to-shallow-clone-a-specific-commit-with-depth-1